### ISSUE TITLE
Some log messages are dissappearing (unicode error occured in logging)

### ISSUE THREAD

**Issue by kumar303** [mozilla/olympia#1362]
jason found this somwehere:
````
'OlympiaWebApp-applogdecoder-1' error: No match: Jan 8 06:59:40 A unicode error occured in logging
````
This likely means that some log messages are not making it to kibana. We should fix this or find a workaround.

**Comment by kumar303** [comment]
if we do https://github.com/mozilla/olympia/issues/1364 then we can probably ignore this unicode error (which is happening in Heka, I think).

**Comment by kewisch** [comment]
Can this one be closed with 1364 fixed?

**Comment by kumar303** [comment]
It would be nice to fix this specific error but, yes, #1364 makes it less of a priority so I'm fine with closing it.
